### PR TITLE
fix maxLength to work as suggested by the jsonschema docs

### PR DESCRIPTION
**Pull request by lucasvmiguel**
This PR fixes the issue #155

### CODE REVIEW

**Comment by lucasvmiguel** [comment]
Could this PR be merged even with `codecov` complaining? 

I made an one line change which has a test already.

**Comment by omissis** [comment]
hi @lucasvmiguel , thanks for the contribution! I will take a look soon and hopefully merge it quickly (don't worry about codecov)

File: tests/validation_test.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ func TestMaxStringLength(t *testing.T) {
 			wantErr: nil,
 		},
 		{
```
Comment (AlbertoBarba) [inline tests/validation_test.go:26]: I'll probably add another test case to check that a value bigger than `maxLenght`returns error as expected

Comment (lucasvmiguel) [reply tests/validation_test.go:26]: Thanks for the suggestion. I've added a test for that 👍

**Comment by lucasvmiguel** [comment]
Hey @omissis 

First of all, thanks for helping maintain this amazing package. It's helping a lot the Go community.

Don't wanna rush you guys, I know that you are probably all busy. But, do you guys know when this PR is gonna get merged? (Can I help with anything else?)

**Comment by omissis** [comment]
hey, I have been sick the past week. I will check this out as soon as I will be up and running again. My estimate would be a week or two. thanks for your patience!

**Review by omissis — APPROVED** [review]

**Pull request merged**